### PR TITLE
final deploy version

### PR DESCRIPTION
**Pull request by duongtnhat**
Final deploy version, same with version on heroku
https://creepy-demon-26908.herokuapp.com/

### CODE REVIEW

**Comment by duongtnhat** [comment]
ready

File: app/models/assign.rb [hunk 1]
```diff
@@ -4,6 +4,6 @@ class Assign < ApplicationRecord
   validates :card, uniqueness: {scope: :user}, presence: true
   validates :user, uniqueness: {scope: :card}, presence: true
   scope :not_assign_yet, ->(project, card) {
-    project.users.where "`users`.`id` NOT IN (?)", card.users.select(:id)
+    project.users.where "\"users\".\"id\" NOT IN (?)", card.users.select(:id)
```
Comment (huuphuc93) [inline app/models/assign.rb]: Dùng where.not đi em

**Comment by huuphuc93** [comment]
commented
comment nhỏ

**Comment by duongtnhat** [comment]
ready

File: app/models/assign.rb [hunk 2]
```diff
@@ -4,6 +4,6 @@ class Assign < ApplicationRecord
   validates :card, uniqueness: {scope: :user}, presence: true
   validates :user, uniqueness: {scope: :card}, presence: true
   scope :not_assign_yet, ->(project, card) {
-    project.users.where "`users`.`id` NOT IN (?)", card.users.select(:id)
+    project.users.where.not "\"users\".\"id\" IN (?)", card.users.select(:id)
```
Comment (huuphuc93) [inline app/models/assign.rb]: project.users.where.not user_id: card.users.pluck(:id)

**Comment by huuphuc93** [comment]
commented

**Comment by duongtnhat** [comment]
ready

**Pull request merged**